modules/core/src/main/scala/org/scalasteward/core/io/WorkspaceAlg.scala

```diff
@@ -17,45 +17,63 @@
 package org.scalasteward.core.io
 
 import better.files.File
-import cats.FlatMap
+import cats.Monad
 import cats.syntax.all._
 import org.scalasteward.core.application.Config
 import org.scalasteward.core.buildtool.BuildRoot
 import org.scalasteward.core.data.Repo
 import org.typelevel.log4cats.Logger
 
 trait WorkspaceAlg[F[_]] {
-  def cleanReposDir: F[Unit]
+  def removeAnyRunSpecificFiles: F[Unit]
 
   def rootDir: F[File]
 
+  def runSummary: F[File]
+
   def repoDir(repo: Repo): F[File]
 
   def buildRootDir(buildRoot: BuildRoot): F[File]
 }
 
 object WorkspaceAlg {
+
+  val RunSummaryFileName: String = "run-summary.md"
+
   def create[F[_]](config: Config)(implicit
       fileAlg: FileAlg[F],
       logger: Logger[F],
-      F: FlatMap[F]
+      F: Monad[F]
   ): WorkspaceAlg[F] =
     new WorkspaceAlg[F] {
       private val reposDir: File =
         config.workspace / "repos"
 
+      private val _runSummary: File =
```

Comment:
As far as I know we use nowhere the `_` prefix. If you rename the method to `runSummaryFile` the name without the underscore would work.